gettsim/parameters/soz_vers_beitr.yaml

```diff
@@ -4,39 +4,57 @@ soz_vers_beitr:
     en: Contribution rate to statutory social insurances
   description:
     de:
+      ges_krankenv: Beitragssätze für gesetzliche Krankenversicherung.
+                    durchschnitt - durchschnittlicher Beitragssatz, bis 2008
+                    allgemein - allgemeiner Satz (§241 SGB V), seit 2009
+                    ermäßigt - ermäßigter Satz (§243 SGB V)
+                    zusatz - durchschnittlicher Zusatzbeitragssatz.
     en:
+      ges_krankenv: Statury Contribution Rates for mandatory health insurance.
+                    durchschnitt - average contribution rate, until 2008
+                    allgemein - general statutory contribution rate, since 2009
+                    ermäßigt - reduced rate
+                    zusatz - average top-up contribution rate
   unit: share
   1984-01-01:
     ges_krankenv:
-      ag: 0.0572
-      an: 0.0572
+      durchschnitt: 0.1144
```

Comment:
Move to `allgemein` until 1992. See, e.g., https://www.aok-bv.de/hintergrund/gesetze/index_15084.html